### ISSUE TITLE
Should not output "Recorded" message when stopping fails 

### ISSUE THREAD

**Issue by real-or-random** [GothenburgBitFactory/timewarrior#280]
Just did the following typo (should have been "15min" instead of  "15 min"):

```
> timew stop 15 min ago                                                                                                                                    
Recorded XXX
  Started 2020-01-30TXX:XX:XX
  Ended              XX:XX:XX
  Total               XX:XX:XX
The current interval does not have the '15' tag.
```

The error message is correct, and the interval was not stopped, which is also correct. 

But the bug here is that there is a "Recorded" message printed, which indicates that tracking XXX has been stopped. That's maybe related to #116.

**Comment by sruffell** [comment]
I noticed something similar while working on #283 where I was seeing:

```shell
$ src/timew start tag3
Recorded tag2
  Started 2020-02-01T03:54:36
  Ended                 55:08
  Total               0:00:32
Note: 'tag3' is a new tag.
Tracking tag3
  Started 2020-02-01T03:55:08
  Current                  08
  Total               0:00:00
Unable to update database.
$ echo $?
255
```
For this to work it is almost like all the "normal" output needs to go to an in memory buffer and only display to the user on a successful exit.

**Comment by sruffell** [comment]
I've implemented this in sruffell/timewarrior@6ce3b5af4 . I wanted to wait until some of the existing PRs work there way into the dev branch before submitting one for this, but hopefully it will make it into the next release.

```shell
$ src/timew start tag1 1day ago
Note: 'tag1' is a new tag.
Tracking tag1
  Started 2020-02-01T16:18:06
  Current         02T16:18:06
  Total              24:00:00
$ src/timew stop 15 min ago
The current interval does not have the '15' tag.
$ src/timew stop 15m ago
The end of a date range must be after the start.
$ src/timew stop 15min ago
Recorded tag1
  Started 2020-02-01T16:18:06
  Ended           02T16:03:21
  Total              23:45:15
$
```
